core/storage/question/gae_models.py

```diff
@@ -591,8 +589,7 @@ def get_question_skill_links_equidistributed_by_skill(
 
         question_count_per_skill = int(
             math.ceil(
-                python_utils.divide( # type: ignore[no-untyped-call]
-                    float(total_question_count), float(len(skill_ids)))))
+                (float(total_question_count) / float(len(skill_ids)))))
```

Comment:
```suggestion
                float(total_question_count) / float(len(skill_ids))))
```
also move this to previous line